### PR TITLE
fix : Show "Thank you/Info" message and redirect to the previous page

### LINKED ISSUE #6471: Sign up: Show "Thank you/Info" message and redirect to the previous page
After signup there is no message what happened for the user and what they have to do next. Therefore please implement the following after the signup process to enter email and password:
1. Show a popup "Thank you for signing up on eventyay.com. We have sent an email with a verification link to you. Please go to your email account and verify your email to get full access to the system. We are logging you into your newly created account now, but your account functionalities are limited until your email is verified. Please press "Ok" to be redirected to the previous page." [OK]
2. When the user presses "Ok" redirect the user to the page they visited before they signed up, e.g. Call for speakers of an event, Frontpage, Public event page of a specific event.
3. Do not offer the option to click the "not verified" message away. The message should always be visible unless the account is verified.
Screenshot from 2021-01-23 23-03-35

### PR DESCRIPTION
**Pull request by ranjitkshah**
Fixes #6471 

#### Short description of what this resolves:
self-explanatory

#### Screenshot:
image

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/register.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ export default class RegisterController extends Controller {
     this.model.save()
       .then(user => {
         this.set('session.newUser', user.get('email'));
+        this.confirm.prompt(this.l10n.t('Thank you for signing up on eventyay.com.'), { hideDeny: true, approveText: this.l10n.t('OK'), extra: this.l10n.t('We have sent an email with a verification link to you. Please go to your email account and verify your email to get full access to the system. We are logging you into your newly created account now, but your account functionalities are limited until your email is verified. Please press "Ok" to be redirected to the previous page.') });
```
Comment (ranjitkshah) [inline app/controllers/register.js:25]: confused here, how to pass this action 
`this.send('loginExistingUser', user.get('email'), password, this.inviteToken, this.event);`
as a argument or is this correct approach? @iamareebjamal

**Comment by ranjitkshah** [comment]
@iamareebjamal, please review

File: app/controllers/register.js [hunk 2]
```diff
@@ -17,12 +17,12 @@ export default class RegisterController extends Controller {
   }
 
   @action
-  createUser() {
+  async createUser() {
```
Comment (iamareebjamal) [inline app/controllers/register.js:20]: No need for async here

**Comment by iamareebjamal** [comment]
Please test what happens if I close the message. It should not show an error

**Comment by ranjitkshah** [comment]
> Please test what happens if I close the message. It should not show an error

https://www.loom.com/share/0a37ab1fcc3d461d814e91c3d5ded35f please check

**Comment by iamareebjamal** [comment]
You clicked on OK

**Comment by ranjitkshah** [comment]
> You clicked on OK

how to close this message? there is no close button 😅

**Comment by iamareebjamal** [comment]
Click outside the dialog

**Comment by ranjitkshah** [comment]
> Click outside the dialog

I tried but it's not closing the dialog. It work only with OK button.
please check once again
image

**Pull request merged**